frontend/src/bundles/notifications/notifications-api.ts

```diff
@@ -0,0 +1,94 @@
+import { type Http } from '~/framework/http/types/types.js';
+import { BaseHttpApi } from '~/framework/http-api/http-api.js';
+import { type Storage } from '~/framework/storage/storage.js';
+
+import { NotificationsApiPath } from './enums/enums.js';
+import {
+    type NotificationRequestDto,
+    type NotificationResponseDto,
+} from './types/types.js';
+
+type Constructor = {
+    baseUrl: string;
+    http: Http;
+    storage: Storage;
+};
+
+class NotificationsApi extends BaseHttpApi {
+    public constructor({ baseUrl, http, storage }: Constructor) {
+        super({ path: NotificationsApiPath.ROOT, baseUrl, http, storage });
```

Comment:
```suggestion
        super({ path: ApiPath.NOTIFICATIONS, baseUrl, http, storage });
```
Seems like here should be route path